subiquity/cmd/server.py

```diff
@@ -129,6 +129,12 @@ def make_server_args_parser():
         default=".subiquity",
         help="in dryrun, control basedir of files",
     )
+    parser.add_argument(
+        "--no-enhanced-secureboot",
+        dest="enhanced_secureboot",
+        action="store_false",
+        default=True,
+    )
```

Comment:
Can't you say `parser.add_argument("--enhanced-secureboot", action="store_true", default=True)` to get the same effect? (i.e. doesn't argparse add --no-$blah options automatically for boolean options?)

Reply:
`action=argparse.BooleanOptionalAction` is the magic

Reply:
py3.9.  I'm going to use focal-compatible syntax rather than mess with integration tests for this.